quadhoral-openmeteo/conftest.py

```diff
@@ -16,7 +17,7 @@ def clean_valid_coord():
 @pytest.fixture
 def valid_json_data():
     '''Returns an example data from API'''
-    return [{'hourly': {'temperature_2m': [15], 
+    return [{'hourly': {'temperature_2m': [15],
```

Comment:
Cya later whitespace, I hope you're happy pylint

Reply:
Enjoy the money pylint.